main.tf

```diff
@@ -24,14 +32,27 @@ resource "aws_route53_zone" "default" {
     "$${stage}", module.this.stage),
     "$${id}", module.this.id),
     "$${attributes}", join(module.this.delimiter, module.this.attributes)),
+<<<<<<< HEAD
     "$${parent_zone_name}", coalesce(join("", data.aws_route53_zone.parent_zone.*.name), var.parent_zone_name, "none")),
   "$${region}", data.aws_region.default.name)
+=======
+    "$${parent_zone_name}", coalesce(join("", data.aws_route53_zone.parent_zone.*.name), var.parent_zone_name)),
+  "$${region}", join("", data.aws_region.default.*.name))
+
+  dynamic "vpc" {
+    for_each = var.private_hosted_zone_vpc_attachments
+    content {
+      vpc_id     = vpc.value.vpc_id
+      vpc_region = vpc.value.vpc_region
+    }
+  }
+>>>>>>> kevcube/support_private_hosted_zone
```

Comment:
Merge issue

Reply:
My bad, happened when I was cleaning up the initial merge pull. Resolved.